### DISCUSSION TITLE
Extruder motor spins backwards Ender 3 V2

### DISCUSSION THREAD

**Discussion by Across32** [mriscoc/Ender3V2S1#727] [Q&A] [answered]
My extruder motor spins backwards, works in reverse. I used the original cable but the extruder motor does not work with it. I changed the cable but the new cable is works in reverse. How to solve it? Thanks.

**Comment by deck55** [comment, marked as answer]
From a terminal you can send:
C562 E0 ; No inverted
Or
C562 E1 ; Inverted

Not by printer right now, but I would expect there is this setting under advance options as well.

**Reply by mriscoc** [reply]
Advanced / Filament Settings / Invert Extruder  

![filament-settings2](https://user-images.githubusercontent.com/2745567/218320430-9c2e4c60-b9a6-4f00-bc90-d5b891a0ae34.jpg)

**Comment by Across32** [comment]
Thank you. Work with Invert Extruder setting under Filament Settings. Thank you.
